mysql/README.md.jinja

```diff
@@ -13,19 +13,6 @@
 - [Usage](#usage)
 - [Metrics](#metrics)
 
-=======
-{% import "macros.jinja" as macros %}
-
-{% if target == "docs" -%}
-# MySQL
-
-- [Overview](#overview)
-- [Setup](#setup)
-- [Built-in Content](#built-in-content)
-- [Usage](#usage)
-- [Metrics](#metrics)
-
->>>>>>> dcb68c5fffabac207e0a84b514a416e0161627e7:mysql/SMART_AGENT_MONITOR.md
```

Comment:
was this deleted  to address a merge conflict with master?

Reply:
yes